manager/controlapi/config.go

```diff
@@ -0,0 +1,264 @@
+package controlapi
+
+import (
+	"bytes"
+	"regexp"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/identity"
+	"github.com/docker/swarmkit/log"
+	"github.com/docker/swarmkit/manager/state/store"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+)
+
+// Currently this is contains the unimplemented config functions in order to satisfy the interface
```

Comment:
Heh, I remember removing it for secrets, but I guess I didn't remove it for configs.